### PR TITLE
Feature/remove core

### PR DESCRIPTION
**Pull request by marioncottard**

### CODE REVIEW

File: README.rst
Comment (ggsdc) [inline README.rst:1]: You can now add the badges for cornflow to see if they work

**Review by ggsdc — APPROVED** [review]

**Comment by ggsdc** [comment]
Fix the conflicts now...

**Review by ggsdc — APPROVED** [review]

**Comment by ggsdc** [comment]
@marioncottard it should be ready to merge right?

**Comment by marioncottard** [comment]
Now it is, yes

**Pull request merged**